.vib/jaeger/vib-verify.json

```diff
@@ -0,0 +1,66 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/jaeger"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/jaeger"
+        },
+        "runtime_parameters": "cXVlcnk6CiAgc2VydmljZToKICAgIHR5cGU6IExvYWRCYWxhbmNlcgogICAgcG9ydHM6CiAgICAgIGFwaTogODAKYWdlbnQ6CiAgc2lkZWNhcnM6CiAgICAtIG5hbWU6IGhvdHJvZC1leGFtcGxlCiAgICAgIGltYWdlOiBqYWVnZXJ0cmFjaW5nL2V4YW1wbGUtaG90cm9kOjEuMzkKICAgICAgYXJnczoKICAgICAgICAtIGFsbAogICAgLSBuYW1lOiB0cmFjZS1nZW5lcmF0b3IKICAgICAgaW1hZ2U6IGJpdG5hbWkvYml0bmFtaS1zaGVsbDoxMS1kZWJpYW4tMTEtcjU3CiAgICAgIGNvbW1hbmQ6CiAgICAgICAgLSAvYmluL2Jhc2gKICAgICAgICAtIC1lYwogICAgICAgIC0gfAogICAgICAgICAgIyEvYmluL2Jhc2gKICAgICAgICAgIHdoaWxlIHRydWU7IGRvCiAgICAgICAgICAgICMgU2FtcGxlIHRha2VuIGZyb20gaW5zcGVjdGluZyBob3Ryb2QgY29kZQogICAgICAgICAgICBjdXJsICdodHRwOi8vMTI3LjAuMC4xOjgwODAvZGlzcGF0Y2g/Y3VzdG9tZXI9MzkyJm5vbnNlPTAuNDU4Mzg4ODQ3MTA1MDE3MjYnCiAgICAgICAgICAgIHNsZWVwIDUKICAgICAgICAgIGRvbmUK",
```

Comment:
You are checking in goss changes in containerPorts and other elements, you need to ensure that the runtime_parameters include those changes

Reply:
Updated [837b47c](https://github.com/bitnami/charts/pull/13480/commits/837b47c8f108e3ef23d7baf76cc4342b17852703)